### ISSUE TITLE
PHP notice when changing admin page's theme

### ISSUE THREAD

**Issue by John-Shaft** [Piwigo/Piwigo#747]
Hi,

I got this notice in my Apache error log when switching the theme of the admin page : 

```
[Sun Aug 27 16:49:21.736536 2017] [proxy_fcgi:error] [pid 30996:tid 121883778676480] [client 2001:db8:3849::42] AH01071: Got error 'PHP message: PHP Notice:  Only variables should be passed by reference in /path/admin.php on line 65\n', referer: https://piwigo.example.org/path/admin.php?change_theme=1
```

I'm running Apache 2.4.25 with php7.0-fpm and PHP 7.0.22. Piwigo is up-to-date

Don't look like a big issue, but then again I'm lacking PHP knowledge and it pollutes my error log :)

**Comment by plegall** [comment]
Fixed on #774 commit https://github.com/Piwigo/Piwigo/commit/943ab9d1899368624eab924fab59f8af708f9418 by @MaximeBOURMAUD

**Comment by plegall** [comment]
... and cherry-picked in 2.9 https://github.com/Piwigo/Piwigo/commit/6530720d5112fb9a99fe3f3b7c3b8bd720774fd2
